src/pages/Dashboard/Versions/Detail/index.jsx

```diff
@@ -0,0 +1,471 @@
+        isOpen={isDialogOpen}
+        onSubmit={this.deleteVersion}
+        onCancel={hideModal}
+      >
+        {t('Delete Version desc')}
+      </Dialog>
+    );
+  };
+
+  renderNoteDialog = () => {
+    const { appVersionStore, t } = this.props;
+    const { isDialogOpen, hideModal } = appVersionStore;
+
+    return (
+      <Dialog
+        title={t('Note')}
+        isOpen={isDialogOpen}
+        onSubmit={() => this.handleVersion('submit', true)}
+        onCancel={hideModal}
+        okText={t('Submit review')}
+        cancelText={t('Continue testing')}
+      >
+        <div className={styles.noteContent}>
+          <p>
+            在正式提交审核之前，请确认你的应用已经通过了以下基本功能的测试：
+          </p>
+          <div className={styles.noteList}>
+            1. 成功部署实例<br />
+            2. 集群运行正常<br />
+            3. 增加节点正常<br />
+            4. 删除节点正常 <br />
+            5. 修改节点配置正常 <br />
+            6. 切换私有网络正常<br />
+            7. 暂停集群正常<br />
+            8. 删除集群正常<br />
+            9. 恢复集群正常<br />
+          </div>
+          <p>
+            更全面的测试手册请参看<Link to={'#'}>《开发者测试指南》</Link>。
+          </p>
+        </div>
+      </Dialog>
+    );
+  };
+
+  renderReason(record) {
+    const { t } = this.props;
+
+    if (!record.message) {
+      return null;
+    }
+
+    return (
+      <div className={styles.reason}>
+        {t('Reason')}:
+        {record.isExpand ? (
+          <div>
+            <pre>{record.message}</pre>
+            <br />
+            <div
+              onClick={() => this.toggleReason(record)}
+              className={styles.expand}
+            >
+              {t('Collapse')}
+            </div>
+          </div>
+        ) : (
+          <label>
+            <span className={styles.hideReason}>&nbsp;{record.message}</span>
+            <span
+              onClick={() => this.toggleReason(record)}
+              className={styles.expand}
+            >
+              {t('Expand')}
+            </span>
+          </label>
+        )}
+      </div>
+    );
+  }
+
+  renderTestInstance() {
+    const { t } = this.props;
+
+    return (
+      <Card className={styles.testInstance}>
+        <div className={styles.title}>{t('Test instance')}</div>
+        <div className={styles.note}>
+          {t('There are no deployment test instances in this release')}
+        </div>
+      </Card>
+    );
+  }
+
+  renderLatestRecord() {
+    const { appVersionStore, userStore, t } = this.props;
+    const { users } = userStore;
+    const { version, audits } = appVersionStore;
+    const audit = _.get(audits[version.version_id], '[0]', {});
+
+    return (
+      <Card className={styles.latestRecord}>
+        <div className={styles.title}>{t('Latest record')}</div>
+        <Status
+          type={audit.status}
+          name={audit.status}
+          className={styles.status}
+        />
+        <div className={styles.record}>
+          <div className={styles.operator}>
+            {t(audit.role)}:&nbsp;{
```

Comment:
Better get rid of `&nbsp;`, if you want style an element, using className to wrap it.